build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -202,7 +202,7 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
                 List<string> cmakeArgList = new List<string>();
 
                 string cmakeBaseRid, visualStudio, archMacro, arch;
-                string ridMacro = $"-DCLI_CMAKE_RUNTIME_ID:STRING={rid}";
+                string ridMacro = $"-DCLI_CMAKE_RUNTIME_ID:STRING=win10-{platform.ToLower()}";
```

Comment:
In general, the change is fine. That said, I would suggest that this "RIDForBuildingHost" be setup in BuildContext in PrepareTargets.Init so that other parts of the build can access it if required. This ensures central initialization of such concepts instead of them being spilled all over the codebase.

And you can, then, fetch it from the BuildContext and use it.